src/scalarstats.jl

```diff
@@ -234,15 +234,23 @@ sem{T<:Real}(a::AbstractArray{T}) = sqrt(var(a) / length(a))
 
 # Median absolute deviation
 """
-    mad(v, center=median(x); constant=1.4826)
+    mad(v)
 
-Compute the median absolute deviation of `v`, optionally specifying a
-precomputed median `center`. `constant` provides a scaling factor that
-defaults to 1.4826 for consistent estimation of the standard deviation
-of a normal distribution.
+Compute the median absolute deviation of `v`.
 """
-mad{T<:Real}(v::AbstractArray{T}, args...;arg...) = mad!(copy(v), args...;arg...)
-mad{T<:Real}(v::Range{T}, args...;arg...) = mad!([v;], args...;arg...)
+function mad{T<:Real}(v::AbstractArray{T})
+    m = median(v)
```

Comment:
This change is not reflected in the documentation. I'd like to change the center to `mean` but kept running into MethodErrors. There is also not a path for those of us that would like to use `mean` as the center.